Added blog post with StrimziCon 2026 schedule (#570)
strimzi/strimzi.github.io: _posts/2026-04-17-strimzicon2026-schedule.md

---
layout: post
title:  "StrimziCon 2026: Schedule announced!"
date: 2026-04-17
author: paolo_patierno
---

We are very pleased to announce the speakers and sessions for the third edition of StrimziCon!
The interest from the community was really high and we received a lot of awesome proposals.
The program committee found it really tough to choose among them in order to build an amazing agenda.

<!--more-->

![StrimziCon 2026 Banner](/assets/images/posts/2026-02-18-strimzicon2026-banner.png)

### Awesome sessions ... join StrimziCon! 

> The conference will take place on <b>Wednesday, June 3rd, 2026</b> (afternoon CEST).

Reviewer: why HTML tags and not markdown formatting?

> It will start with a keynote from Kate Stanley (IBM, Strimzi maintainer) and Paolo Patierno (IBM, Strimzi maintainer).

Reviewer: ```suggestion
The event opens with a keynote from Kate Stanley and Paolo Patierno (IBM, Strimzi maintainers).
```

> The keynote will be followed by almost 4 hours of breakout sessions covering several topics from core Strimzi features, to use cases, operations and integrations.

Reviewer: ```suggestion
The keynote is followed by almost 4 hours of breakout sessions with topics covering core Strimzi features, use cases, operations, and integrations.
```

> | Time (CEST) | Session |

Reviewer: ```suggestion
| Time (CEST) | Session title |
```

Author: I would prefer stick with just "Session" as previous years.

|:---:|:---:|
| 14:00 - 14:20 | Keynote (Kate Stanley & Paolo Patierno @ IBM) |
| 14:25 - 14:55 | Beyond the Install: Take full ownership of your Strimzi cluster (Jakub Scholz @ Cloudera) |
| 15:00 - 15:30 | Who Are You? Configuring Kafka Authentication in Strimzi from TLS to Custom Principals (Daniel Mulder @ Axual) |
| 15:35 - 16:05 | Swapping the Engine Mid-Flight: How We Moved Reddit’s Petabyte Scale Kafka Fleet to Kubernetes (Sky Kistler @ Reddit) |
| 16:10 - 16:40 | Disaster Recovery in action with Kafka and Strimzi (Mickael Maison @ IBM) |
| 16:45 - 17:15 | From Running to Operating: Real-World Strimzi in Production (Rajith Attapattu @ Randoli) |
| 17:20 - 17:50 | Building Strimzi MCP Server for Kubernetes: Democratizing Platform Expertise Through LLMs (David Kornel, Jakub Stejskal @ IBM) |
| 17:50 - 18:00 | Closing (Kate Stanley & Paolo Patierno @ IBM) |

> You can find more details about speakers and sessions on the official [page](https://community.cncf.io/events/details/cncf-virtual-project-events-hosted-by-cncf-presents-strimzicon-virtual-2026/) as part of the CNCF events.

Reviewer: ```suggestion
For more details about speakers and sessions, see the [CNCF event page](https://community.cncf.io/events/details/cncf-virtual-project-events-hosted-by-cncf-presents-strimzicon-virtual-2026/).
```

> Looking forward to seeing you at the conference and don't forget to [register](https://community.cncf.io/events/details/cncf-virtual-project-events-hosted-by-cncf-presents-strimzicon-virtual-2026/). It's free!

Reviewer: ```suggestion
We look forward to seeing you at the conference. [Register here](https://community.cncf.io/events/details/cncf-virtual-project-events-hosted-by-cncf-presents-strimzicon-virtual-2026/). It's free!
```
